src/main/java/org/codehaus/plexus/archiver/AbstractArchiver.java

```diff
@@ -108,9 +109,9 @@
     private boolean useJvmChmod = true;
 
     /**
-     * @since 4.2.0
+     * @since 4.3.0
      */
```

Comment:
I wonder why we actually need a `@since` on private fields. I see them necessary on public methods only.